packages/appstore/src/components/main-title-bar/regulators-switcher.tsx

```diff
@@ -57,6 +60,6 @@ const RegulatorSwitcher = () => {
             )}
         </div>
     );
-};
+});
 
-export default observer(RegulatorSwitcher);
+export default RegulatorSwitcher;
```

Comment:
Why thou? isn't it cleaner to keep the observer here?

Reply:
According to the documentation, [https://mobx.js.org/react-integration.html#always-read-observables-inside-observer-components](url). Its to reduce re-renders and optimising the performance

Reply:
@matin-deriv @thisyahlen-deriv We will lose the component type, That's why we need to wrap the component definition itself with `observer` instead of the reference to the component definition 🙇🏻 

<img width="1551" alt="image" src="https://user-images.githubusercontent.com/72082844/233282217-35e51589-8415-4337-8ece-bb365dc06cf0.png">

<img width="1551" alt="image" src="https://user-images.githubusercontent.com/72082844/233282371-29433fb6-fe74-430c-84a6-86d8af48ed88.png">

Reply:
Alright, but I can see that it is being widely used all over the code base 🤔

Reply:
We are in the midst in refactoring it in every test 😸